ClientDesktop/UserIndex.xaml.cs

```diff
@@ -18,11 +18,10 @@
 using System.Windows.Shapes;
 
 using User = Models.User;
-using ClientMVC.Models;
 
 namespace ClientDesktop
 {
-    public partial class UserIndex : Page
+    public partial class UserIndex : Window
     {
         public UserIndex()
```

Comment:
Instead of that search by ID field, you can turn this into a exact copy of the "Show" page in the MVC client. The window constructor can receive parameters, such as a User object. Use that object to display the Skills list of that specific User. To reach this window, link to it from the Index page or something, passing a User as a parameter.
[https://stackoverflow.com/questions/20547904/wpf-passing-string-to-new-window](url)